papers/doc/6-sample/README.md

```diff
@@ -1,3 +1,29 @@
-# sample editor
+# samples
 
-TODO
+In the context of Furnace, a sound sample (usually just referred to as a sample) is a string of numbers that hold sampled PCM audio.
+
+In Furnace, these samples can be generated by importing a .wav (think of it as an higher quality MP3) file.
+
+## supported systems
+
+As of Furnace 0.5.5, the following sound chips have sample support:
+ - NES/Ricoh 2A03 (PCM only, no DPCM, and only on channel 5)
+ - Sega Genesis/YM2612 (channel 6 only; but only if there exists a `1701` effect that gets played on or before a trigger for a sample)
+ - PC Engine/TurboGrafx 16/Huc6280 (same conditions as above)
+ - Amiga/Paula (on all channels AND resamplable, but you need to make an instrument with the Amiga format and tie it to a sample first)
+ - Arcade/SEGA PCM (same as above but you don't need to make an instrument for it and you have to use the `20xx` effect command to resample your samples)
+ - Neo Geo/Neo Geo EXT-Ch2 (on the last 5 channels only and can be resampled the same way as above)
+ - Theoretically, the GameBoy channel 3 and the main channels of the Huc6280 can play (low quality) samples if data is given to them fast enough.
```

Comment:
Currently Furnace does not support samples on Game Boy wave channel. I suggest this line be removed.